osu.Game/Overlays/Mods/ModButton.cs

```diff
@@ -151,8 +151,8 @@ public Mod Mod
         [BackgroundDependencyLoader]
         private void load(AudioManager audio)
         {
-            sampleOn = audio.Sample.Get(@"Checkbox/check-on");
-            sampleOff = audio.Sample.Get(@"Checkbox/check-off");
+            sampleOn = audio.Sample.Get(@"UI/check-on");

```

Comment:
We should probably move all these filenames into a separate file as constants so that we don't repeat the same string multiple times throughout the codebase.